Controller/Command.java

```diff
@@ -0,0 +1,85 @@
+package Controller;
+
+public class Command {
+    public char getAction(String[] command) {
+        switch (command[0]) {
+            case "a":
+                return 'a';
+            case "m":
+                return 'm';
+            case "s":
+                return 's';
+            case "h":
+                return 'h';
+            default:
+                return '#';
+        }
+    }
+    public String checkArgs(String[] command, char action) {
+        int argsCount = command.length;
```

Comment:
See below. If we make changes to make this object initialization this logic will go into each of the different objects.